frontend/src/pages/ProfilePage/Profile/container.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import Presenter from './presenter';
+
+const Profile = () => {
+  const [isLoading, setLoading] = useState(true);
+  const [username, setUsername] = useState('');
+  const [userId, setUserID] = useState(-1);
+  const [kakaoId, setKakaoId] = useState('');
+  const [phone, setPhone] = useState('');
+  const [bio, setBio] = useState('');
+
+  const targetUrl = '/account/user/1';
```

Comment:
Don't hardcode this, instead get the current userId somehow and pass it there instead. We need to implement #50 anyway, might as well go ahead with it.